File: InformacjeTurystyczne/InformacjeTurystyczne/Models/Tabels/Shelter.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace InformacjeTurystyczne.Models.Tabels
{
    public class Shelter
    {
        [Key]
        [Display(Name = "Klucz schronisko")]
        public int IdShelter { get; set; }

        [Display(Name = "Nazwa schroniska")]
        public string Name { get; set; }

        [Display(Name = "Max il. miejsc")]
        public int MaxPlaces { get; set; }

        [Display(Name = "Miejsca")]
        public int Places { get; set; }

        [Display(Name = "Otwarte")]
        public bool IsOpen { get; set; }

        [Display(Name = "Numer telefonu")]
        public string PhoneNumber { get; set; }

        [Display(Name = "Opis")]
        public string Description { get; set; }

        [Display(Name = "Klucz obcy region")]
        public int IdRegion { get; set; }

        [Display(Name = "Region")]
        public virtual Region Region { get; set; }

        public virtual ICollection<PermissionShelter> PermissionShelters { get; set; }
    }
}

File: InformacjeTurystyczne/InformacjeTurystyczne/wwwroot/js/shelter.js
import Util from "./utility.js";

const Shelter = function () {
    this.type = "schronisko";
    this.name = "N/A";
    this.region = "N/A";
    this.places = "N/A";
    this.maxPlaces = "N/A";
    this.open = "N/A";
    this.phoneNumber = "N/A";
    this.description = "N/A";
}

Shelter.prototype.render = function () {
    let itemDiv = Util.createElement("div", { withClass: "info__item" });

    itemDiv.appendChild(Util.createElement("h1", { withText: this.name }));
    itemDiv.appendChild(Util.createElement("p", { withText: this.type, withClass: "info__type" }));
    itemDiv.appendChild(Util.createRow("region:", this.region));
    itemDiv.appendChild(Util.createRow("wolne miejsca:", this.places));
    itemDiv.appendChild(Util.createRow("wszystkich miejsc:", this.maxPlaces));
    itemDiv.appendChild(Util.createRow("otwarte:", this.open));
    itemDiv.appendChild(Util.createRow("telefon:", this.phoneNumber));
    itemDiv.appendChild(Util.createRow("opis:", this.description));

    return itemDiv;
}

export default Shelter;